src/__tests__/exampleData.js

```diff
@@ -219,4 +219,107 @@ export const action = deepFreeze({
     email: selfAccount.email,
     apiKey: selfAccount.apiKey,
   },
+  realm_init: {
+    type: REALM_INIT,
+    data: {
+      last_event_id: 34,
+      msg: 'msg',
+      queue_id: 1,
```

Comment:
Yikes this is long :astonished: But that's how the reality is, so that's cool.

How did you get this particular set of values -- was it the result of some real request? ... Ah, no, that can't be in the case of `realm_uri` and some others.

One thing that doesn't matter in this test but I think would be helpful: use `selfAccount.realm` and similar values, to make this fit together consistently with the other example data.